src/common/colors.ts

```diff
@@ -1,8 +1,21 @@
-export default {
-  teal: '#24dcbe',
-  white: '#fff',
-  lightGrey: '#f9f9f9',
-  grey: '#F2F3F5',
-  mediumGrey: '#D8D8D8',
-  darkBlue: '#00a1d8',
+const TEAL = '#24DCBE';
```

Comment:
I don't think you need to all-caps const names, could check with a style guide to be sure?

Reply:
Generally i've been using uppercase for these constants that are getting used globally...like the ones in `injectGlobals` except i wanted to keep the colours in a separate file.
https://google.github.io/styleguide/jsguide.html#naming-constant-names and https://www.javascripttutorial.net/es6/javascript-const/

Reply:
ok as long as there is some sort of rule for them and not random ... I think JS style is usually non all-caps but if we have a system we should just stick to it

Reply:
heh well i've been using this in the gdc for awhile so not sure if that qualifies as a system...but I do like it since it seems more clearly a constant to me.